torch/fx/README.md

```diff
@@ -132,3 +140,6 @@ def forward(self, x):
 ```
 
 This is the core of why FX is a Python-to-Python translation toolkit. Outside users can treat the results of FX transformations as they would any other `nn.Module` instance.
+
+# Next steps
+If you're interested in learning more about obtaining fx graphs, which kinds of IRs are available to you and how to execute simple transformations make sure to check out [this tutorial](https://colab.research.google.com/drive/1Zh-Uo3TcTH8yYJF-LLo5rjlHVMtqvMdf)
```

Comment:
lol... thanks for promoting this.... 
I am on the hook of keeping this up-to-date....